Language/Functions/Advanced IO/pulseInLong.adoc

```diff
@@ -15,7 +15,7 @@ subCategories: [ "Advanced I/O" ]
 === Descrição
 `pulseInLong()` é uma alternativa à função link:../pulsein[pulseIn()], sendo melhor para lidar com pulsos longos e situações afetadas por interrupções.
 
-Reads a pulse (either HIGH or LOW) on a pin. For example, if value is HIGH, `pulseInLong()` waits for the pin to go `HIGH`, starts timing, then waits for the pin to go `LOW` and stops timing. Returns the length of the pulse in microseconds or 0 if no complete pulse was received within the timeout.
+Lê um pulso (ALTO ou BAIXO) em um pino. Por exemplo, se o valor for HIGH, `pulseInLong ()` espera que o pino vá `HIGH`, começa a cronometrar e, em seguida, espera que o pino vá` LOW` e pára de cronometrar. Retorna a duração do pulso em microssegundos ou 0 se nenhum pulso completo foi recebido dentro do tempo limite.
```

Comment:
```suggestion
Lê um pulso (HIGH ou LOW) em um pino. Por exemplo, se o valor for HIGH, `pulseInLong()` espera que o pino vá `HIGH`, começa a cronometrar e, em seguida, espera que o pino vá `LOW` e pára de cronometrar. Retorna a duração do pulso em microssegundos ou 0 se nenhum pulso completo foi recebido dentro do tempo limite.
```

- Code words `LOW` and `HIGH` should not be translated.
- Remove spaces that were inserted into code markup